### PR TITLE
Port hpc-stack to S4

### LINKED ISSUE #258: Port to S4
To support the port of the global workflow to SSEC's S4 cluster, hpc-stack should be ported and installed on that system.

### PR DESCRIPTION
**Pull request by DavidHuber-NOAA**
Added `config_s4.sh` and the new module file `hpc-miniconda.lua`, modeled after `hpc-miniconda3.lua`, to address #258.

The official install of hpc-stack on S4 is located on `/data/prod/hpc-stack` and can be accessed with `module use /data/prod/hpc-stack/modulefiles/stack`.

closes #258

### CODE REVIEW

**Review by kgerheiser — APPROVED** [review]

**Comment by kgerheiser** [comment]
Looks good. Thanks for the port.

File: config/config_s4.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+export MAKE_VERBOSE=N
+export   MAKE_CLEAN=N
+export DOWNLOAD_ONLY=N
+export STACK_EXIT_ON_FAIL=Y
+export WGET="wget -nv"
+
+# Load these basic modules for S4
+module purge
+module load license_intel/S4
+module load git/2.30.0
```
Comment (aerorahul) [inline config/config_s4.sh:24]: I think you will need to add a line like this:
```
module use /path/to/miniconda/3.8-s4/modulefile
```
to be able to load the miniconda/3.8-s4  modulefile unless it is being installed via hpc-stack, which I don't think we do, yet.

**Review by aerorahul — APPROVED** [review]
Looks good.
I made one comment in `config_s4.sh` to check if that is what you mean.

File: config/config_s4.sh (hunk 1 above)
Comment (DavidHuber-NOAA) [reply config/config_s4.sh:24]: On S4, the path to miniconda/3.8-s4 is in `$MODULEPATH` by default, so it's good to go.  Thanks for checking.

**Pull request merged**